README.rst

```diff
@@ -64,6 +64,13 @@ following RFC-4752, section 3.1:
 
     import winkerberos as kerberos
 
+    def get_channel_binding_struct(response):
```

Comment:
Is there a practice as to how large this docstring should be? I can add the standard `tls-server-end-point:` one and links to the RFC if needed.